src/components/sponsors.js

```diff
@@ -1,25 +1,29 @@
 function Sponsors() {
+  let numbers = 1;
+
   const sponsorList = [
     '../../static/images/desktop/UCRlogo.png',
     '../../static/images/desktop/ExCITElogo.jpg',
   ];
   // initial render
   return (
+
     <div id="sponsors" className='grid-container'>
       <div className="column-20 pre-2 leader-3 trailer-3">
         <h1>Our Sponsors</h1>
         <p>The work at <strong>IESD</strong> is made
          possible by the following sponsors</p>
         <div className="sponsor-gallery">
           {sponsorList.map((sponsor) => (
-            <div className="column-6">
-              <img src={sponsor}/>
+            <div key={numbers.toString()} value={numbers++} className="column-6">
```

Comment:
@Andymendez100 when working with `map` you can also use the index parameter. So try something like `sponsorList.map((sponsor, index) =>...` and set the `key` to the index, something like `key={index}` for example. It's much easier and straightforward.

Thanks for working on this!

Reply:
The documentation of react they said they don’t recommend using the index

Reply:
@Andymendez100 Kudos for reading! However, that case is valid when lists change, this one is static for the time the user is on the page, the react documentation (the good word) says the following as well, Chapter 20, verse 15 -> 'We don’t recommend using indexes for keys if the order of items **may change**. This can negatively impact performance and may cause issues with component state' - so its fine to use. We use it at Mozilla for the same purpose.

Reply:
So should i change it to use index?

Reply:
Right. You're essentially doing the same thing as the index key. We're safe just using the index as the key since this won't change dynamically with user interaction or other changes.